contracts/NFTVesting/BasicNFTTimelock.sol

```diff
@@ -28,16 +30,19 @@ contract BasicNFTTimelock {
 
     /**
      * @dev Deploys a timelock instance that is able to hold the token specified, and will only release it to
-     * `beneficiary_` when {release} is invoked after `releaseTime_`. The release time is specified as a Unix timestamp
+     * `beneficiary_` when {release} is invoked after `releaseTime_`. The cliff period is specified as a Unix timestamp
      * (in seconds).
      */
     constructor(
-        IERC20 nft_,
+        IERC721 nft_,
         uint256 tokenId_,
         address beneficiary_,
         uint256 releaseTime_
     ) {
-        require(releaseTime_ > block.timestamp, "BasicNFTTimelock: release time is before current time");
+        require(
+            releaseTime_ > block.timestamp,
+            "BasicNFTTimelock: cliff period is before current time"
```

Comment:
`cliff period` -> `release time`?

Reply:
The function body is inserted where the special symbol "_;" appears in the definition of a modifier. So its calling the rest of the function with `_;`

Reply:
thanks for the catch